src/Blocks/custom/result-output/components/result-output-editor.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { select } from '@wordpress/data';
+import { ServerSideRender,
+	checkAttr,
+	icons,
+	AsyncSelect,
+	getAttrKey,
+	STORE_NAME,
+} from '@eightshift/frontend-libs/scripts';
+import { __ } from '@wordpress/i18n';
+import { Placeholder } from '@wordpress/components';
+import { getFilteredAttributes, outputFormSelectItemWithIcon } from '../../../components/utils';
+
+export const ResultOutputEditor = ({
+	attributes,
+	setAttributes,
+	formSelectOptions,
+	resultSelectOptions,
+}) => {
+	const manifest = select(STORE_NAME).getBlock('result-output');
+
+	const {
+		blockFullName
+	} = attributes;
+
+	const {
+		attributesSsr,
+	} = manifest;
+
+	const resultOutputFormPostId = checkAttr('resultOutputFormPostId', attributes, manifest);
+	const resultOutputFormPostIdRaw = checkAttr('resultOutputFormPostIdRaw', attributes, manifest);
+	const resultOutputPostIdRaw = checkAttr('resultOutputPostIdRaw', attributes, manifest);
+	const resultOutputPostId = checkAttr('resultOutputPostId', attributes, manifest);
+
+	if (resultOutputPostId?.length < 1 || resultOutputFormPostId?.length < 1) {
+		return (
+			<Placeholder
+					icon={icons.form}
+					label={<span className='es-font-weight-400'>{__('Eightshift Forms - Result output', 'productive')}</span>}
```

Comment:
```suggestion
					label={<span className='es-font-weight-400'>{__('Eightshift Forms - Result output', 'eightshift-forms')}</span>}
```
Pls search-replace this string in project